highfidelity/hifi
# Case 20339 - nitpick v1.3.1

https://highfidelity.fogbugz.com/f/cases/20339/
# Description
1.  Online help has been implemented
2.  All entities within a 1000 m radius are deleted by nitpick before a test run.
     This is implemented by running Interface with `deleteNearbyEntities.js` script

# Testing
Tests are to be performed on both Windows and Mac
1.  Run nitpick
2.  Click Help->Online readme
3.  Verify that the nitpick readme.md file has opened in the default browser
4.  Run the full test suite.
